### ISSUE TITLE
Narrator does not read the correct line in debugger editor of F12

### ISSUE THREAD

**Issue by JargoonPard** [microsoft/monaco-editor#478]
monaco-editor version: 0.X.Y
Browser: Edge Microsoft Edge
OS: Windows 10
Steps or JS usage snippet reproducing the issue:
Using the files in the attached zip I was able to reproduce this easily. 
Steps:
1. Browse to ReaderTest.html and open F12 Tools (F12)
2. In the Debugger tab set the breakpoint as indicated in the screenshot. (menu.min.js)
3. click the "Add Menu Item" button in the web page
4. Turn on Narrator (Win+Ctrl+Enter)
5. Click on the "Debugger" tab in F12
6. Tab until you enter the debugger menu
7. Use the arrows keys to navigate to the step into button
8. Press `Enter` to activate the Step into action
9. Narrator does not start reading at any of the expected locations.

[ReaderTest.zip](https://github.com/Microsoft/monaco-editor/files/1099169/ReaderTest.zip)

**Comment by alexandrudima** [comment]
I think the problem here is that the F12 tools do not set the cursor position inside the monaco editor when stepping to the same position where the breakpoint hit. I will try to contact the F12 team, but I don't think this is necessarily a Monaco Editor issue.

**Comment by alexandrudima** [comment]
Turns out that this is caused by the default behaviour of Narrator + Edge when focusing a `<textarea>`.

Minimal repro:
```html
<!DOCTYPE html>
<html>
<head>
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
</head>
<body>

<textarea id="test" style="width:800px;height:200px;"></textarea>
<button id="button">Click me</button>

<script>
var lines = [
    '  return (d.value = a), b.attributes.setNamedItem(d), (b.innerText = a), b;',
    '}',
    'function AddMenuItem(a) {',
    '  let b = createNewMenuEntry(a);',
    '  const c = document.getElementById(`theMenu`);',
    '  c.appendChild(b);',
    '}',
    'function createNewMenuEntry(a) {',
    '  let b = document.createElement("li"), c = document.createAttribute("role");',
    '  (c.value = "menuitem"), b.attributes.setNamedItem(c);',
];

document.getElementById('button').onclick = function(e) {
    e.preventDefault();

    function offsetOfPosition(line, col) {
        var result = 0;
        for (var i = 0; i < line - 1; i++) {
            result += lines[i].length + 1 /*LF*/;
        }
        result += col - 1;
        return result;
    }
    // Place cursor on line 4, column 1
    var offset = offsetOfPosition(4, 1);
    var text = lines.join('\n');

    var ta = document.getElementById('test');
    ta.value = text;
    ta.focus();
    ta.setSelectionRange(offset, offset);
};

</script>
</body>
</html>

```

**Comment by alexdima** [comment]
Closing, as Microsoft Edge Legacy has reached end of support on March 9, 2021. See [the end of support for Microsoft Edge Legacy announcement](https://docs.microsoft.com/en-us/lifecycle/announcements/edge-legacy-eos-details)
